docs/samples/Microsoft.ML.Samples/Transformers.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.StaticPipe;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.ML.Samples
+{
+    public static class Transformers
+    {
+
+        public class Input
+        {
+            public float Feature0 { get; set; }
+            public float Feature1 { get; set; } 
+            public float Feature2 { get; set; }
+            public float Feature3 { get; set; }
+            public float Target { get; set; }
+        }
+
+        public static IEnumerable<Input> GetInputData()
+        {
+            var data = new List<Input>();
+            data.Add(new Input { Feature0 = -2.75f, Feature1 = 0.77f, Feature2 = -0.61f, Feature3 = 0.14f, Target = 140.66f });
+            data.Add(new Input { Feature0 = -0.61f, Feature1 = -0.37f, Feature2 = -0.12f, Feature3 = 0.55f, Target = 148.12f });
+            data.Add(new Input { Feature0 = -0.85f, Feature1 = -0.91f, Feature2 = 1.81f, Feature3 = 0.02f, Target = 402.20f });
+
+            return data;
+        }
+
+        /// <summary>
+        /// The example for the staticlly typed concat estimator
+        /// </summary>
+        public static void ConcatEstimator()
+        {
+            // Create a new environment for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var env = new LocalEnvironment();
+
+            IEnumerable<Input> data = GetInputData();
+
+            //A preview of InputData:
+            //feature_0; feature_1; feature_2; feature_3;  target
+            //-2.75;     0.77;     -0.61;      0.14;       140.66
+            //-0.61;    -0.37;     -0.12;      0.55;       148.12
+            //-0.85;    -0.91;      1.81;      0.02;       402.20
+
+            //convert to an DataView
```

Comment:
Space and capital letter, would be better.

Reply:
and "." in the end of sentence. 
Heil Mein Grammar!

---
In reply to: [223520620](https://github.com/dotnet/machinelearning/pull/1189#discussion_r223520620) [](ancestors = 223520620)

Reply:
Proper grammar is ... overhead!

---
In reply to: [223521069](https://github.com/dotnet/machinelearning/pull/1189#discussion_r223521069) [](ancestors = 223521069,223520620)